Frontend/src/components/Table/Row.svelte

```diff
@@ -13,6 +14,33 @@
   };
 </script>
 
+<tr
+  on:click
+  style={`--rowHeight: ${rowHeight}px; ${
+    rowBackgroundColorFunction
+      ? "background-color: " + rowBackgroundColorFunction(item)
+      : ""
+  }`}>
+  {#each columns as col}
+    <td>
+      {#if col.isImageUrl}
+        {#if item[col.key] && displayValue(col, item) !== ""}
+          <img src={displayValue(col, item)} alt="item" />
+        {/if}
+      {:else}
+        <div
+          class="cell"
+          style={cellBackgroundColorFunction
+            ? "background-color: " + cellBackgroundColorFunction(col, item)
```

Comment:
When you make cellBackroundColorFunction return a promise you need to await the result here like so https://svelte.dev/tutorial/await-blocks